### DISCUSSION TITLE
Html y CSS del Login

### DISCUSSION THREAD

**Discussion by veroivk** [FrancoGL/ISPC-FullStack#36] [General]
He tratado de mejorar el html y css correspondientes al login. Mis compañeros del meet del sabado 15 de octubre me ayudaron a solucionar algunos errores. Se me complicó bastante poder resolver el responsive. Traté cambiando el diseño pero he vuelto al anterior. Seguiré tratando de mejorarlo. Lamentablemente este sprint 1 cierra el 17 de octubre.

**Comment by ZafkielEdt** [comment]
Ya veo el tema del responsive, veré si puedo hacerlo como mencionaron ayer.

**Reply by veroivk** [reply]
Yo estoy viendo otra opción para ver cómo queda. También tratando de localizar el error/es en el actual login que impide que sea responsive.

**Reply by veroivk** [reply]
Perdón no había visto que ya lo arreglaste!!!
